corehq/motech/openmrs/tasks.py

```diff
@@ -192,51 +191,54 @@ def import_patients_to_domain(domain_name, force=False):
     for importer in get_openmrs_importers_by_domain(domain_name):
         if not _should_import_today(importer) and not force:
             continue
+        import_patients_with_importer.delay(importer.to_json())
 
+@task(serializer='pickle', queue='background_queue')
+def import_patients_with_importer(importer_json):
+        importer = OpenmrsImporter.wrap(importer_json)
         password = b64_aes_decrypt(importer.password)
-        requests = Requests(domain_name, importer.server_url, importer.username, password)
+        requests = Requests(importer.domain, importer.server_url, importer.username, password)
```

Comment:
might be better to just have `domain=importer.domain` on top or may be pass it as an argument to the method itself.